apps/meteor/app/lib/server/functions/parseUrlsInMessage.ts

```diff
@@ -1,21 +1,23 @@
+import type { IMessage } from '@rocket.chat/core-typings';
+
 import { Markdown } from '../../../markdown/server';
-import { getMessageUrlRegex } from '../../../../lib/getMessageUrlRegex.ts';
+import { getMessageUrlRegex } from '../../../../lib/getMessageUrlRegex';
 
-export const parseUrlsInMessage = (message) => {
+export const parseUrlsInMessage = (message: IMessage & { parseUrls?: boolean }): IMessage => {
 	if (message.parseUrls === false) {
 		return message;
 	}
 
 	message.html = message.msg;
 	message = Markdown.code(message);
 
-	const urls = message.html.match(getMessageUrlRegex()) || [];
+	const urls = message.html?.match(getMessageUrlRegex()) || [];
 	if (urls) {
-		message.urls = [...new Set(urls)].map((url) => ({ url }));
+		message.urls = [...new Set(urls)].map((url) => ({ url, meta: {} }));
```

Comment:
`meta: {}`?

Reply:
It's for "metadata", required in the urls type, but not being passed here 👀 

I can make it optional, aa seems to not cause any problems not being there lol